code/modules/reagents/reagent_containers/food/drinks.dm

```diff
@@ -1454,7 +1454,6 @@
 			src.reagents.reaction(get_turf(src), TOUCH) //splat the floor AND the thing we hit, otherwise fuel wouldn't ignite when hitting anything that wasn't a floor
 			if(hit_atom != get_turf(src)) //prevent spilling on the floor twice though
 				src.reagents.reaction(hit_atom, TOUCH)  //maybe this could be improved?
-			spawn(5) src.reagents.clear_reagents()  //maybe this could be improved?
```

Comment:
:thinking:

Reply:
It hasn't worked for years because reagents is null by the time it reaches that spawn. What's the point of clearing reagents if reagents itself is getting deleted?

Reply:
No point. This exact string of code is copypasted in several other places.